src/cms/templatetags/form_icon_filters.py

```diff
@@ -0,0 +1,20 @@
+import ntpath
```

Comment:
```suggestion
import os
```
I propose to use the standard os package instead of the windows-style ntpath.